OOP/task_01.py

```diff
@@ -0,0 +1,10 @@
+class Car:
+    def __init__(self, make, model, year):
+        self.make = make
+        self.model = model
+        self.year = year
+    def show_info(self):
+        print(f"Make: {self.make} \nModel: {self.model} \nYear: {self.year}")
```

Comment:
Check also attribute difference if we create a class like this

```suggestion
class Car:
    make = ""
    model = ""
    year = 0

toyota = Car()
toyota.make = "Toyota"
toyota.model = "Camry"
toyota.year = 2020
```